App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
```

Comment:
do we need this file?

Reply:
I did not open this file becuase I have no work with this file